### PR TITLE
week one HW added

### PR DESCRIPTION
**Pull request by Vahablotfi**

### CODE REVIEW

File: week1/weekOne-HW/Program.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    char[] charArray = input.ToCharArray();
+    Array.Reverse(charArray);
+    return new string(charArray);
+});
+
+
+
+//2. String/Math
+
+app.MapGet("/Exercise2", (string input) => {
```
Comment (Vertygo) [inline week1/weekOne-HW/Program.cs:17]: I like the fact that you have used query parameter for this 👍

File: week1/weekOne-HW/Program.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+app.MapGet("/Exercise3", () => {
+    int[] arr = new[] { 271, -3, 1, 14, -100, 13, 2, 1, -8, -59,  -1852, 41, 5 };
+    int sumOfNegatives = 0;
+    int multiOfPositive =1;
+    for (int i = 0; i < arr.Length; i++)
+    { 
+        if (arr[i]<= 0)
+        {
+            sumOfNegatives -= arr[i];
```
Comment (Vertygo) [inline week1/weekOne-HW/Program.cs]: Sum of negatives should give negative value but this is ok too 👍

File: week1/weekOne-HW/Program.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    } 
+    else
+    {
+          do
+    {
+         fibonacciNumbers.Add(fibonacciNumbers[fibonacciNumbers.Count - 1] + fibonacciNumbers[fibonacciNumbers.Count - 2]);
+    } while (fibonacciNumbers.Count<=input);
+  
+    };
+      return $"{input} fibonacci number is :{fibonacciNumbers[fibonacciNumbers.Count - 1]}";
```
Comment (Vertygo) [inline week1/weekOne-HW/Program.cs]: The formatting here is a bit off. You can use VSCode formatter from C# plugin to fix this (key combination is ALT+SHIFT+F on windows).

**Review by Vertygo — APPROVED** [review]

File: week1/weekOne-HW/Program.cs (hunk 2 above)
Comment (Vahablotfi) [reply week1/weekOne-HW/Program.cs]: I fixed it thank you .

**Pull request merged**